scripts/pre-commit-smart-checks.sh

```
@@ -1,334 +1,17 @@
+# Back-compat entry: single hook for downstream repos that pin `specfact-smart-checks`.
+# Canonical layout is modular hooks in .pre-commit-config.yaml → pre-commit-quality-checks.sh.
+#
+# Resolves the quality script from the repository root so copies under .git/hooks/pre-commit work.
+set -euo pipefail
+
+_script_path=${BASH_SOURCE[0]}
+case "${_script_path}" in
+  /*) ;;
+  *) _script_path=$(pwd)/${_script_path} ;;
+esac
+_hook_dir=$(CDPATH= cd -- "$(dirname "${_script_path}")" && pwd)
```

Shellcheck (0.11.0)

[warning] Remove space after = if trying to assign a value (for empty string, use var='' ... ).

(SC1007)